apps/agent/test/agent_shared.js

```diff
@@ -45,6 +45,19 @@ module.exports = (
 
     let ETH, ANY_ENTITY, APP_MANAGER_ROLE, EXECUTE_ROLE, SAFE_EXECUTE_ROLE, RUN_SCRIPT_ROLE, ADD_PROTECTED_TOKEN_ROLE, REMOVE_PROTECTED_TOKEN_ROLE, ADD_PRESIGNED_HASH_ROLE, DESIGNATE_SIGNER_ROLE, ERC1271_INTERFACE_ID
 
+    // Web3 1.x inverts the order of the parameters of the sign function:
+    // Web3 0.x => web3.eth.sign(address, dataToSign, [, callback])
+    // Web3 1.x => web3.eth.sign(dataToSign, address [, callback])
+    // As a temporary solution, we don't use the helper and define it here instead.
+    web3Sign = async (signer, hash) => {
+      return new Promise((resolve, reject) => {
```

Comment:
It sounds like we should just migrate to the web3@1 version of this function in contract-helpers.

Do we need to wrap it around a promise at the moment because web3@0.x didn't return a promise?